.changeset/moody-trainers-repair.md

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
Just playing devils advocate here. There is already a heartbeatTask kicked off in the StorageTaskBroker class kicked off when the TaskManager is created.

Can / should we expect that the taskmanager is picking up the task and kicking off the heartbeat within the 2 minutes (or whatever the nginx timeout is)?